project.clj

```diff
@@ -4,7 +4,20 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.9.0-alpha17" :scope "provided"]
+                 [org.clojure/clojurescript "1.9.908"]
```

Comment:
Could add `:scope "provided"` to this? 

(Or alternatively move both clojure & clojurescript dependencies to dev profile.)